qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/CreateDeploymentTest.java

```diff
@@ -132,11 +137,19 @@ public void shouldRejectDeployIfProcessIsInvalid() {
   @Test
   public void shouldRejectDeployIfResourceIsTooLarge() {
     // when
+    final var modelThatFitsJustWithinMaxMessageSize =
+        Bpmn.createExecutableProcess("PROCESS")
+            .startEvent()
+            .documentation(
+                "x"
+                    .repeat(
+                        (int) (ByteValue.ofMegabytes(MAX_MSG_SIZE_MB) - ByteValue.ofKilobytes(2))))
+            .done();
```

Comment:
🎉 Always nice to see less bpmn files. You scared me with the 12k lines removed though